apps/frontend/src/components/layout/Header/Header.view.tsx

```diff
@@ -1,99 +1,54 @@
 
 import classes from './Header.module.scss';
 
 type TProps = {
-	readonly isMenuOpen: boolean;
-	readonly isMenuVisible: boolean;
-	readonly theme?: string;
-	readonly float?: boolean;
-	readonly fromNavMenu?: boolean;
-	readonly onToggleMenu: () => void;
-	readonly onCloseMenu: (linkName: string) => void;
+	readonly logout: () => void;
 };
-
 const HeaderView = (props: TProps) => {
-	const { route } = useRouter();
-
-	const containerClass = concatClasses(
-		classes,
-		'container',
-		props.isMenuOpen ? 'container--slideUp' : '',
-		props.float ? 'container--float' : '',
-		route.includes('contact') ? 'container--contact' : '',
-	);
-
-	const menuClass = concatClasses(
-		classes,
-		'container__menu',
-		props.isMenuOpen || props.fromNavMenu ? 'container__menu--active' : '',
-		props.theme === 'dark' ? 'container__menu--dark' : 'container__menu--light',
-		props.fromNavMenu ? 'container__menu--fromNavMenu' : '',
-		route !== '/contact' ? 'container__menu--hover' : '',
-	);
-
-	const logoClass = concatClasses(
-		classes,
-		'container__logo',
-		props.theme === 'dark' ? 'container__logo--dark' : 'container__logo--light',
-		props.fromNavMenu ? 'container__logo--fromNavMenu' : '',
-		props.isMenuOpen ? 'container__logo--inMenu' : '',
-	);
-
-	const textClass = concatClasses(
-		classes,
-		'container__text',
-		props.theme === 'dark' ? 'container__text--dark' : 'container__text--light',
-		props.isMenuOpen ? 'container__text--inMenu' : '',
-	);
-
-	const mobileBurgerContainerClasses = concatClasses(
-		classes,
-		'mobileBurger',
-		props.fromNavMenu ? 'mobileBurger--fromNavMenu' : '',
-	);
-
-	const mobileBurgerIconClasses = concatClasses(
-		classes,
-		'mobileBurger__icon',
-		props.isMenuOpen ? 'mobileBurger__icon--active' : '',
-		props.theme === 'dark' ? 'mobileBurger__icon--dark' : 'mobileBurger__icon--light',
-	);
+	const isAuth = useSelector((state: RootState) => state.user.isAuthenticated);
 
 	return (
-		<header className={containerClass}>
-			<div className={mobileBurgerContainerClasses}>
-				<button
-					className={classes['mobileBurger__button']}
-					type="button"
-					onClick={props.onToggleMenu}
-				>
-					<Svg name="humburger" className={mobileBurgerIconClasses} />
+		<div className={classes['container']}>
+			<NavLink to="/" className={classes['navLink']}>
+				<img src={Logo} alt="logo" className={classes['container__image']} />
+			</NavLink>
+			{!isAuth && (
+				<div className={classes['linksContainer']}>
+					<NavLink
+						to="/auth/login"
+						className={({ isActive }) =>
+							`${classes['linksContainer__link']} ${
+								isActive ? classes['linksContainer__link--active'] : ''
+							}`
+						}
+					>
+						LOGIN
+					</NavLink>
+					<NavLink
+						to="/auth/register"
+						className={({ isActive }) =>
+							`${classes['linksContainer__link']} ${
+								isActive ? classes['linksContainer__link--active'] : ''
+							}`
+						}
+					>
+						REGISTER
+					</NavLink>
```

Comment:
you can move all that code:

```
({ isActive }) =>
							`${classes['linksContainer__link']} ${
								isActive ? classes['linksContainer__link--active'] : ''
							}`
``` 

to a function above, always keep the HTML part clean of logic as much as possible.

Also keep "what to display?" logic in `view` files